x-pack/solutions/observability/plugins/apm/public/components/shared/environment_select/index.tsx

```diff
@@ -69,49 +58,36 @@ export function EnvironmentSelect({
     }
   };
 
-  const { data, status: searchStatus } = useFetcher(
-    (callApmApi) => {
-      return isEmpty(searchValue)
-        ? Promise.resolve({ terms: [] })
-        : callApmApi('GET /internal/apm/suggestions', {
-            params: {
-              query: {
-                fieldName: SERVICE_ENVIRONMENT,
-                fieldValue: searchValue,
-                serviceName,
-                start,
-                end,
-              },
-            },
-          });
-    },
-    [searchValue, start, end, serviceName]
```

Comment:
I don't think removing it is a good idea! What would happen if a user has more environments than the limit defined on the search query? 

I was talking to @kpatticha about it and she mentioned that there are clusters with lots of environment configured.

IMHO we should try fixing the current problem without removing this functionality.